### PR TITLE
Security email notifications

### PR DESCRIPTION
**Pull request by ibrechin**

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
retest this please

File: mtp_api/settings/base.py [hunk 1]
```diff
@@ -306,6 +307,19 @@
 
 INVOICE_NUMBER_BASE = 1000000
 
+NOMS_OPS_NOTIFICATIONS_URL = os.environ.get(
+    'NOMS_OPS_NOTIFICATIONS_URL',
+    'https://noms-ops.moneytoprisoners.dsd.io/security/notifications/'
```
Comment (ushkarev) [inline mtp_api/settings/base.py]: i'd prefer the fallbacks to be localhost:8003

File: mtp_api/templates/notification/periodic_email.txt [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{% load i18n %}
+
+{% blocktrans trimmed with name=user.first_name %}
+  Dear {{ name }},
```
Comment (ushkarev) [inline mtp_api/templates/notification/periodic_email.txt]: all lines should be de-dented

**Pull request merged**